examples/ansible/roles/alp_reboot/tasks/end.yml

```diff
@@ -0,0 +1,19 @@
+- name: Remove the myplaybook.service service file
+  become: true
+  ansible.builtin.file:
+    path: /etc/systemd/system/myplaybook.service
+    state: absent
+
+- name: Remove the myplaybook.service service file
+  become: true
+  ansible.builtin.file:
+    path: "/home"
+    state: touch
```

Comment:
This makes no sense.

Reply:
@pdostal, thank you for your review. My apologies for the confusion. That play was indeed intended for testing purposes and was mistakenly included in the commit. I will remove it and push the changes shortly.

Reply:
@pdostal can you please elaborate on your "too much at once" comment. I see the PR as primarily adding the adding auto-reboot functionality and I think and in general any new functionality would ideally be done in a single PR.

I do see changes in setup_libvirt_host.ym that are more style changes,  like the changes to single lines on some of the "when:" statements and could/should have been done in a different PR. Is that what you meant?